NuGet.config

```diff
@@ -2,6 +2,8 @@
 <configuration>
   <packageSources>
     <add key="AspNetVNext" value="https://www.myget.org/F/aspnetcidev/api/v3/index.json" />
+    <add key="dotnet-cli" value="https://www.myget.org/F/dotnet-cli/api/v3/index.json" />
```

Comment:
@pranavkm is going to make it so that we don't reference other NuGet/MyGet feeds. Please make sure these don't get checked in.